.github/workflows/CI.yml

```diff
@@ -0,0 +1,122 @@
+name: CI
+# Run on master, tags, or any pull request
+on:
+  schedule:
+    - cron: '0 2 * * *'  # Daily at 2 AM UTC (8 PM CST)
+  push:
+    branches: [master]
+    tags: ["*"]
+  pull_request:
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - 1
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
+        include:
+          # Add a 1.0 job just to make sure we still support it
+          - os: ubuntu-latest
+            version: 1.0.5
+            arch: x64
+          # Add a 1.3 job because that's what Invenia actually uses
+          - os: ubuntu-latest
+            version: 1.3
+            arch: x64
+          # Add a 32-bit job to ensure we don't have any 64-bit specific logic
+          - os: ubuntu-latest
+            version: 1
+            arch: x86
+          # Arm64 currently not supported by julia-actions/setup-julia
+          ## Note: Conda.jl cannot be used on ARM currently which means we have to use a system Python install
```

Comment:
Can we make an issue for this to come back to it at a future time?

Reply:
#25